egs/sre16/v2/run.sh

```diff
@@ -178,10 +181,9 @@ if [ $stage -le 3 ]; then
   # wasteful, as it roughly doubles the amount of training data on disk.  After
   # creating training examples, this can be removed.
   local/nnet3/xvector/prepare_feats_for_egs.sh --nj 40 --cmd "$train_cmd" \
+    --write_utt2num_frames true \
```

Comment:
Usually the options look like this: --write-utt2num-frames

Reply:
My bad. Let me change it.